bin/compile

```diff
@@ -27,15 +27,23 @@ download(){
 	DB="$CACHE_DIR/$EDITION.mmdb"
 
 	if [ -f $TAR ]; then
-    modified=$(curl --silent --head $URL | \
-                 awk '/^[Ll]ast-[Mm]odified/{print $0}' | \
-                 sed 's/^[Ll]ast-[Mm]odified: //')
-    echo "-----> $EDITION: modified $modified"
-    remote_ctime=$(date --date="$modified" +%s)
-    local_ctime=$(stat -c %y "$TAR")
-    local_ctime=$(date --date="$local_ctime" +%s)
-    if [ $local_ctime -lt $remote_ctime ]; then
-      newer_version=true
+    response=$(curl --silent --head $URL | tee \
+                      >(awk '/HTTP\/2 /{print $0}') \
+                      >(awk '/^[Ll]ast-[Mm]odified/{print $0}') \
+                      > /dev/null )
+    code=$(echo "$response" | grep 'HTTP\/2' | sed 's/^HTTP\/2 //')
+    if [[ "$code" != *"200"* ]]; then
+      echo "WARNING $EDITION: Unable to reach Maxmind, received: $code"
```

Comment:
The ticket says we're supposed to log "what the maxmind API returns" — your thinking here is that just the HTTP status code of the response is sufficient? Is this something where if we're debugging this failure in the future, we'll have other information to go off of besides this failed request, so all we really need to know is the code?

Reply:
yeah so its configured to curl twice, the first being a silent curl to just check the date of the latest version. if thats newer, we do a second curl to get the latest version. i chose to preserve the silent first curl and just pull out the status code as thats likely sufficient to debug what the underlying issue is for this simple functionality. I'm not super fluent in what all curl configurations exist, but I thought that if we wanted to pull the error message, we'd have to get the full body of the zip each time

Reply:
makes sense! and it wouldn't be that hard to make the request again while debugging, and just logging the error code is a lot simpler than trying to anticipate whatever else they might include in the response.